packages/next/build/utils.ts

```diff
@@ -536,6 +537,13 @@ export async function buildStaticPaths(
     )
   }
 
+  if (typeof staticPathsResult.fallback !== 'boolean') {
+    throw new Error(
+      `The \`fallback\` key must be returned from unstable_getStaticProps in ${page}.\n` +
```

Comment:
`unstable_getStaticProps`? Isn't it "unstable_getStaticPaths"?

Reply:
Fixed, thanks!

Reply:
Fixed, thanks!